### PR TITLE
Made it possible to override `entrypoint`

### PR DESCRIPTION
**Pull request by brurucy**
Quite often you have an image, and you want to change the ENTRYPOINT.

This is a very tiny change, all that I did was to create a new option.

### CODE REVIEW

**Comment by brurucy** [comment]
@orlangure I have tested it locally for side-effects, and it seems to work just fine.

**Comment by orlangure** [comment]
Hi @brurucy and thank you so much for your contribution!

I like the idea, but unfortunately I have a very busy schedule this week, so I will be able to take a closer look, merge the PR and prepare a new release a little bit later. I really hope to do that until the end of this week.

Thank you!

File: gnomock_test.go [hunk 1]
```diff
@@ -191,6 +191,35 @@ func TestGnomock_withCommand(t *testing.T) {
...
 }
 
+func TestGnomock_withEntrypoint(t *testing.T) {
+	r, w := io.Pipe()
+
+	container, err := gnomock.StartCustom(
+		testutil.TestImage,
+		gnomock.DefaultTCP(testutil.GoodPort80),
+		gnomock.WithLogWriter(w),
+		gnomock.WithEntrypoint("/app"),
```
Comment (orlangure) [inline gnomock_test.go:201]: I think that this test is not enough, it doesn't prove that `WithEntrypoint` actually works. To prove it, I think we should have 2 subtests: one that does exactly what you suggest, and the other one uses a wrong entrypoint, such as `echo`, and expects to get an error message like `echo: command not found`. What do you think?

Comment (brurucy) [reply gnomock_test.go]: Sure @orlangure , I have just changed it to be as you described.

**Review by orlangure — APPROVED** [review]
Fantastic, thank you!

**Pull request merged**